lib/src/rules/avoid_unused_constructor_parameters.dart

```diff
@@ -54,11 +54,12 @@ class _ConstructorVisitor extends RecursiveAstVisitor {
   final Set<FormalParameter> unusedParameters;
 
   _ConstructorVisitor(this.rule, this.element)
-      : unusedParameters = element.parameters.parameters
-            .where((p) =>
-                p.declaredElement is! FieldFormalParameterElement &&
-                !p.declaredElement.hasDeprecated)
-            .toSet();
+      : unusedParameters = element.parameters.parameters.where((p) {
+          final element = p.declaredElement;
+          return element is! FieldFormalParameterElement &&
+              !element.hasDeprecated &&
+              element.name != '_';
```

Comment:
Don't we want to allow any name that consists entirely of underscores? Otherwise we're limiting users to at most one ignored unused parameter.

Reply:
Absolutely. Good catch.

Reply:
Analyzer's `lib/src/lint/util.dart` has `isJustUnderscores` for this... deprecated though...